packages/astro/src/integrations/astroFeaturesValidation.ts

```diff
@@ -131,15 +131,15 @@ function validateAssetsFeature(
 		isSquooshCompatible = false,
 	} = assets;
 	if (config?.image?.service?.entrypoint === SHARP_SERVICE && !isSharpCompatible) {
-		logger.error(
+		logger.warn(
 			'astro',
 			`The currently selected adapter \`${adapterName}\` is not compatible with the image service "Sharp".`
 		);
 		return false;
 	}
 
 	if (config?.image?.service?.entrypoint === SQUOOSH_SERVICE && !isSquooshCompatible) {
-		logger.error(
+		logger.warn(
```

Comment:
the user shouldn't need to make any manual changes; therefore, an error is too severe